test/openmic_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require './lib/open_mic'
+
+class OpenMicTest < Minitest::Test
+
+  def setup
+    @open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+  end
+
+  def test_it_exists
+    assert_instance_of OpenMic, @open_mic
+  end
+
+  def test_it_initializes
+    assert_equal "Comedy Works", @open_mic.location
+    assert_equal "11-20-18", @open_mic.date
+    assert_equal [], @open_mic.performers
+  end
+
+  def test_you_can_welcome_performers
+    sal = User.new("Sal")
```

Comment:
These could be moved into the setup method to speed things up and keep this test more DRY.